packages/showcase/examples/responsive-vis/responsive-bar-chart.js

```diff
@@ -19,7 +19,7 @@
 // THE SOFTWARE.
 
 import React from 'react';
-import {AreaSeries, HorizontalBarSeries, XAxis, XYPlot, YAxis} from 'index';
+import {AreaSeries, HorizontalBarSeries, XAxis, XYPlot, YAxis}from 'react-vis';
```

Comment:
prettier?

Reply:
for some reason prettier doesn't seem to be working.  I can look into this later.